contrib/coco/scripts/make_dataset.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+#
+#  Copyright 2021 Rikai Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""Create a test dataset from COCO datasets
+"""
+
+import argparse
+import json
+from itertools import islice
+
+from pycocotools.coco import COCO
+
+
+def build(args):
+    coco = COCO(args.annotation_json)
+    with open(args.annotation_json) as fobj:
+        d = json.load(fobj)
+
+    images = []
+    annotations = []
+    polygon_images = 0
+    rle_images = 0
+    for image_id in islice(coco.imgs, 0, 200):
+        img = coco.loadImgs(image_id)
+        anns = coco.loadAnns(coco.getAnnIds(imgIds=image_id))
+        if any([ann["iscrowd"] for ann in anns]):
+            rle_images += 1
+            if rle_images > 10:
+                break
```

Comment:
nit: though most of the time this , this is dependent on your sampling distribution. How about changing this to continue and add an `if rle_images > 10 and polygon_images > 10: break` below this if-else?

Reply:
So the purpose here is to make the output has 10 RLE and 10 polygons images.

Reply:
right but if you get to 10 RLE first then you won't have 10 polygon images

Reply:
Make sense. Fixed in the code actually.